src/ng/directive/input.js

```diff
@@ -1126,7 +1126,11 @@ function baseInputType(scope, element, attr, ctrl, $sniffer, $browser) {
     });
   }
 
-  var timeout;
+  var timeout, oldVal;
+  var viewValueUpdated = true, msieInput = msie >= 10 && msie <= 11;
+  if (msieInput && inputType === 'text') {
```

Comment:
Looks like you're correct - I assumed it would have the spinner, but apparently not.